Game/RainMeadow.LoadingHooks.cs

```diff
@@ -226,6 +226,7 @@ private void WorldLoader_ctor(On.WorldLoader.orig_ctor_RainWorldGame_Name_bool_s
         {
             if (OnlineManager.lobby != null)
             {
+                OnlineManager.mePlayer.isActuallySpectating = false;
```

Comment:
I think it's best here so we cover death / karma, or its usage in arena. lmk what you think.

Reply:
@youbitchoc pinging you for awareness prior to merge.

Reply:
we set session init books elsewhere, iinw it's in getorinitiatesavestate

Reply:
I'll look and see. I personally think we should set this as high up the stack as possible to prevent any unwanted problems arising

Reply:
we can also shove it in StoryGameSession_ctor

Reply:
My only concern with storyGameSess is if other game modes want to use it at some point.